### PR TITLE
Fix typos in plugin documentation

### PR DESCRIPTION
**Pull request by philippwiesemann**
Thanks!

### CODE REVIEW

File: doc/plugins.dox [hunk 1]
```diff
@@ -98,7 +98,7 @@ at https://github.com/geany/geany-plugins.
 Beside of adding a new plugin, geany-devel-mailing list is also the place where
```
Comment (elextr) [inline doc/plugins.dox]: Doesn't need the "where"

File: doc/plugins.dox [hunk 2]
```diff
@@ -98,7 +98,7 @@ at https://github.com/geany/geany-plugins.
...
 to discuss development related questions.
 However, once you have done your fork of geany-plugins you can develop
-your plugin until you think its the right time to publish it. At this point,
+your plugin until you think it's the right time to publish it. At this point,
```
Comment (elextr) [inline doc/plugins.dox]: personally I would prefer "it is"

**Comment by elextr** [comment]
One grammar I noticed while reading and one comment, rest LGTM.

**Comment by philippwiesemann** [comment]
The commit was updated to include both suggestions.

**Pull request merged**